src/promptflow/promptflow/_cli/_utils.py

```diff
@@ -389,11 +389,14 @@ def get_secret_input(prompt, mask="*"):
     - Ignore control characters and print warning message.
     """
     if not isinstance(prompt, str):
-        raise TypeError(f"prompt must be a str, not ${type(prompt).__name__}")
+        e = TypeError(f"prompt must be a str, not ${type(prompt).__name__}")
+        raise UserErrorException(str(e), privacy_info=[]) from e
```

Comment:
why empty privacy_info here?

Reply:
I remember you want to use None to determine the behavior here. But it looks strange for devs not knowing that.

Reply:
Let's use error format for this case?

Reply:
Because I want to record messages, I need **privacy_info**  is not **None**.
![image](https://github.com/microsoft/promptflow/assets/23182548/7524c669-307c-4882-af10-0e08a441c55f)

Reply:
I will change to using **message_format**, but it still feels strange because **message parameters** are not needed.
![image](https://github.com/microsoft/promptflow/assets/23182548/f4feb09f-fc72-46f8-bfbf-02530bcde37c)